tests/tests/operation_tests.cpp

```diff
@@ -6481,78 +6476,49 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
             proxy.proxy = "alice";
 
             signed_transaction tx;
-            tx.set_expiration(db->head_block_time() + STEEMIT_MAX_TIME_UNTIL_EXPIRATION);
-            tx.operations.push_back(proxy);
-            tx.sign(bob_private_key, db->get_chain_id());
-            db->push_transaction(tx, 0);
-
+            push_tx_with_ops(tx, bob_private_key, proxy);
```

Comment:
I know that the function already contains BOOST_CHECK_NO_THROW, but I would prefer to specify this check explicitly in tests. So the test looks clearer. I would have thought of having one function `push_tx_with_ops` without check instead of two functions `push_tx_with_ops` & `push_tx_with_ops_throw`, and specifying BOOST_CHECK_NO_THROW check in the test itself.